File: task1.py
from flask import Flask, request

app = Flask(__name)

@app.route('/')
def index():
    # Get the value of the 'arg1' query parameter from the user's request
    arg1 = request.args.get('arg1', 'No argument provided')
    
    # Return the argument value as a response
    return f'arg1: {arg1}'

if __name__ == '__main__':
    app.run(debug=True)

File: task1.js
const express = require('express');
const app = express();
const port = 3000;

// Set up a route to handle incoming requests
app.get('/', (req, res) => {
  // Get the value of the 'arg1' query parameter from the request
  const arg1 = req.query.arg1;

  // Check if 'arg1' is provided in the query string
  if (arg1) {
    res.send(`Value of 'arg1': ${arg1}`);
  } else {
    res.send('Please provide a value for "arg1" in the query string.');
  }
});

// Start the server
app.listen(port, () => {
  console.log(`Server is running on http://localhost:${port}`);
});
